image/descriptor.go

```diff
@@ -32,6 +33,10 @@ type descriptor struct {
 	Size      int64  `json:"size"`
 }
 
+func (d *descriptor) normalizeDigest() string {
+	return strings.Replace(d.Digest, ":", "-", -1)
+}
```

Comment:
I see, but `oci-image-tool` isn't working at all w/o this fix and your #159 it's a rather bigger PR with a broader scope, I'd love to have this merged so people can use this tool instead of waiting on #159